api/v1alpha1/podtypes/access_policy.go

```diff
@@ -22,11 +26,15 @@ type OutboundPolicy struct {
 	External []ExternalRule `json:"external,omitempty"`
 }
 
+// If user adds both Namespace and NamespacesByLabel to an InternalRule, Namespace takes presedence
+// +kubebuilder:object:generate=true
```

Comment:
this is _super_ nitpicky, but this sort of sounds like internal documentation and not documentation for the API

maybe some more natural wording would be along the lines of "Adding both Namespace and NamespacesByLabel to InternalRule will cause Namespace to take precedence, and NamespacesByLabel will be omitted"